### ISSUE TITLE
Links to hosted OVA files do not display in Markdown for README.md

### ISSUE THREAD

**Issue by jcrew99** [SamuraiWTF/samuraiwtf#178]
Hi there,

The links to the hosted ova files found in option 1 and 2 in the markdown readme are not displaying correctly. (tested with latest firefox and chrome)

This is what it looks like:
![image](https://user-images.githubusercontent.com/37361441/211150207-adcc5936-5729-4751-af94-55467c6f371e.png)

While it's find-able by looking at the page source, it did take me a second to realise what was going on.

I got curious as to what's going on and noticed that the reference doesn't display due to the double "" in the div style

i.e.
`
[<div style="text-align: center;"> Download SamuraiWTF for Hyper-V</div>](https://downloads-samuraiwtf-com.s3.us-west-2.amazonaws.com/SamuraiWTF_HyperV.zip)
`
Produces: 
[<div style="text-align: center;"> Download SamuraiWTF for Hyper-V</div>](https://downloads-samuraiwtf-com.s3.us-west-2.amazonaws.com/SamuraiWTF_HyperV.zip)

Where
`
[<div style="text-align: center;""> Download SamuraiWTF for Hyper-V</div>](https://downloads-samuraiwtf-com.s3.us-west-2.amazonaws.com/SamuraiWTF_HyperV.zip)
`
Produces:

[<div style="text-align: center;""> Download SamuraiWTF for Hyper-V</div>](https://downloads-samuraiwtf-com.s3.us-west-2.amazonaws.com/SamuraiWTF_HyperV.zip)

So i would think that the double quotations is breaking the context or it's not supported on github markdown.
Either way, can it please be changed to something like this:

[Download SamuraiWTF for Hyper-V](https://downloads-samuraiwtf-com.s3.us-west-2.amazonaws.com/SamuraiWTF_HyperV.zip)

From:
`
[<div style="text-align: center;">Download SamuraiWTF for Hyper-V</div>](https://downloads-samuraiwtf-com.s3.us-west-2.amazonaws.com/SamuraiWTF_HyperV.zip)
`

**Comment by JGillam** [comment]
Huh, strange (because we don't know why they used to display fine and then
stopped sometime recently). Thanks for looking into this.  We will get this
fixed .

**Comment by jcrew99** [comment]
Yeah it's weird, i can't tell why its not working but i'm sure something changed that i haven't come across yet.
Cheers!
